integration_tests/pages/admin/placementApplications/createPlacementPage.ts

```diff
@@ -1,10 +1,18 @@
-import { Cas1PremisesSummary, PlacementRequestDetail } from '../../../../server/@types/shared'
+import {
+  ApprovedPremisesApplication,
+  Cas1PremisesSummary,
+  PlacementRequestDetail,
+} from '../../../../server/@types/shared'
```

Comment:
Import can be shortened:

```suggestion
} from '@approved-premises/api'
```

Reply:
It can indeed. We should probably replace all these relative paths with simpler path aliases at some stage.

Reply:
Yeah, I've only been doing them as and when I find them, I think the IDE does this for me on save, in fact.